docs/guides/elements-lookup-troubleshooting.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Elements Lookup Troubleshooting
+---
+
+This article helps to resolve possible issues that may pop up while looking up for elements with XCUITest driver,
+where the desired element is either not found or not visible in the page source at all.
+
+Since there might be multiple reasons to why an element cannot be found the topic is divided into sections where
+each section contains visible symptoms with the list of their possible resolutions.
+
+
+## Symptom #1
+
+The desired element is shown as part of a bigger container and is not distinguable in the page source tree.
+Sometimes the whole application view with all elements in it is visible as one single container.
+
+## Resolutions To Symptom #1
+
+### Make sure the application under test is accessible
+
+The XCUITest driver is based on Apple's XCTest framework. And the latter uses the information provided by the system
+accessibility framework to interact with on-screen elements, and to distinguish them. The same approach is used by
```

Comment:
It would be worth addressing this accessibility info could be different from Xcode's element inspector. Some issues addressed, "XXX is in the Xcode inspector but not in Appium's source tree," something like this

Reply:
added